.azure-pipelines/test_plan.py

```diff
@@ -130,16 +131,16 @@ def __init__(self):
         super(FinishStatus, self).__init__(TestPlanStatus.FINISHED)
 
 
-def get_scope(elastictest_url):
-    scope = "api://sonic-testbed-tools-dev/.default"
-    if elastictest_url in [
-        "http://sonic-testbed2-scheduler-backend.azurewebsites.net",
-        "https://sonic-testbed2-scheduler-backend.azurewebsites.net",
-        "http://sonic-elastictest-prod-scheduler-backend-webapp.azurewebsites.net",
-        "https://sonic-elastictest-prod-scheduler-backend-webapp.azurewebsites.net"
-    ]:
-        scope = "api://sonic-testbed-tools-prod/.default"
-    return scope
+# def get_scope(elastictest_url):
```

Comment:
should we remove this code other than keep them in the source code with comment out?

Reply:
I'll remove it in subsequent PRs.